how_to_build.md

```diff
@@ -5,9 +5,7 @@ It also replicates the build on the Continuous Integration build server and is t
 
 You can also build the solution using Visual Studio 2019 or later, but this doesn't provide the same assurances as the command line build.
 
-At the time of writing the full build (including all target frameworks) can only run on Windows.
-
-[Partial builds](#building-only-a-subset-of-the-supported-target-frameworks) are supported on Linux (Ubuntu 18.04). This might also run on macOS, but hasn't been tested.
+At the time of writing the full build (including testing on all target frameworks) can only run on Windows.

```

Comment:
I thought we'd test only on .NET Core, actually. But I guess it doesn't hurt to keep testing on .NET Framework as well.

Reply:
I'd be happy to reduce (or keep), but as I think you've mentioned elsewhere, who the heck knows where these analyzers will run, and under what runtime. Like the surprise we got when they stopped working in VS2017 due to the netstandard2.0 update.

Reply:
Anyhow, I see no harm in keeping for now.